packages/vendure-plugin-sort-by-popularity/src/sort-by-popularity.plugin.ts

```diff
@@ -13,6 +15,18 @@ import { SortService } from './sort.service';
     `,
   },
   configuration: (config) => {
+    config.customFields.Product.push({
+      name: 'popularityScore',
+      type: 'float',
```

Comment:
I think we can make the popularity score an int? Let's avoid floats if they are not really needed. We can round the scores to INTs, thats fine.

Reply:
sure I will change it to int